cli/native_deployment/native_deployment.go

```diff
@@ -343,6 +368,17 @@ func (p *NativeDeploymentConfigPlugin) StartTest(homeDirectory string, userInput
 		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, "'%v %v' %v", dev.SERVICE_COMMAND, dev.SERVICE_START_COMMAND, cerr)
 	}
 
+	// Add binds for the provided service secret files to the top-level service container if the secret name is in the service deployment
```

Comment:
Make this a reusable function and call it from the other new function.

Reply:
Also in dev/secrets.go now